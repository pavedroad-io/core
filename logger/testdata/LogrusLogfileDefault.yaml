logpackage: logrus
loglevel: info
enabletimestamps: false
enablecolorlevels: true
enablecloudevents: true
enablekafka: false
kafkaformat: cloudevents
kafkaproducercfg:
  brokers:
  - localhost:9092
  topic: logs
  partition: random
  key: fixed
  keyname: user
  cloudeventsid: hmac
  compression: snappy
  ackwait: local
  flushfreq: 500ns
  enabletls: false
  tlscfg: null
  enabledebug: false
enableconsole: false
consoleformat: text
enablefile: true
fileformat: json
filelocation: testdata/LogrusLogfileDefault.log
