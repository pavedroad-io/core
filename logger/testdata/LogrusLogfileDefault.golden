{"data":"Infof using logrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"info","type":"io.pavedroad.cloudevents.log"}
{"data":"Warnf using logrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"warning","type":"io.pavedroad.cloudevents.log"}
{"data":"Errorf using logrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"error","type":"io.pavedroad.cloudevents.log"}
{"data":"Print usinglogrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"info","type":"io.pavedroad.cloudevents.log"}
{"data":"Printf using logrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"info","type":"io.pavedroad.cloudevents.log"}
{"data":"Println using logrus","source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","subject":"info","type":"io.pavedroad.cloudevents.log"}
